### ISSUE TITLE
shift+4 does

### ISSUE THREAD

**Issue by like101101** [egaga/openvim#95]

**Comment by unelma** [comment]
Yes, instead of shift+4, it's alt+4 for some keyboard layouts, and perhaps something else for others. Without configuring different layouts, this is indeed a problem.

Haven't thought what would be a good option. Thanks for bringing this up.

**Comment by like101101** [comment]
My suggestion is that you guys can provide something like "skip this exercise". So that when we encounter some problems we can still move forward, or in case when we close the page, we don't have to redo all the previous exercises. Even Enter speed up the reading process, we still have to "do the exercise" to move to next section. In my case, shift+4 does not work,  and I can't go further, and when I close and restart, I have to redo all the previous exercise.

**Comment by unelma** [comment]
You can skip to any section from the menu: 
![image](https://user-images.githubusercontent.com/46071946/151859639-a461dfcf-872b-4680-975e-b8cb76e77e56.png)

![image](https://user-images.githubusercontent.com/46071946/151859666-acf0910b-8b81-434e-937f-60bc17d90efb.png)

Hope that helps.

Previously the menu was open by default, but after someone made a pull request, they decided to set it closed by default.

Maybe there could be a skip button, though.

**Comment by like101101** [comment]
Oh cool! Thanks!

**Comment by unelma** [comment]
No problem!
